src/CasparCG.ts

```diff
@@ -1,2365 +1,539 @@
-		return this.do(new AMCP.MixerStatusMastervolumeCommand({ channel: channel }))
-	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-STRAIGHT_ALPHA_OUTPUT>
-	 */
-	public mixerStraightAlphaOutput(channel: number, layer?: number, state?: number | boolean, defer?: boolean): Promise<IAMCPCommand>
-	public mixerStraightAlphaOutput(channel: number, layer?: number, state?: number | boolean, defer?: boolean): Promise<IAMCPCommand> {
-		return this.do(new AMCP.MixerKeyerCommand({ channel: channel, layer: layer, keyer: state, defer: defer }))
-	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-STRAIGHT_ALPHA_OUTPUT>
-	 */
-	public mixerStraightAlphaOutputDeferred(channel: number, layer?: number, state?: number | boolean): Promise<IAMCPCommand> {
-		return this.mixerStraightAlphaOutput(channel, layer, state, true)
-	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-STRAIGHT_ALPHA_OUTPUT>
-	 */
-	public getMixerStatusStraightAlphaOutput(channel: number, layer?: number): Promise<IAMCPCommand> {
-		return this.do(new AMCP.MixerStatusStraightAlphaOutputCommand({ channel: channel, layer: layer }))
-	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-GRID>
-	 */
-	public mixerGrid(channel: number, resolution: number, transitionDuration?: number, transitionEasing?: Enum.Ease | string, defer?: boolean): Promise<IAMCPCommand>
-	public mixerGrid(channel: number, resolution: number, transitionDuration?: number, transitionEasing?: Enum.Ease | string, defer?: boolean): Promise<IAMCPCommand>
-	public mixerGrid(channel: number, resolution: number, transitionDuration?: number, transitionEasing?: Enum.Ease | string, defer?: boolean): Promise<IAMCPCommand> {
-		return this.do(new AMCP.MixerGridCommand({ channel: channel, resolution: resolution, transitionDuration: transitionDuration, transitionEasing: transitionEasing, defer: defer }))
-	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-GRID>
-	 */
-	public mixerGridDeferred(channel: number, resolution: number, transitionDuration?: number, transitionEasing?: Enum.Ease | string): Promise<IAMCPCommand> {
-		return this.mixerGrid(channel, resolution, transitionDuration, transitionEasing, true)
-	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-COMMIT>
-	 */
-	public mixerCommit(channel: number): Promise<IAMCPCommand> {
-		return this.do(new AMCP.MixerCommitCommand({ channel: channel }))
+	async load(params: LoadParameters): Promise<Request> {
+		return this.executeCommand({
+			command: Commands.Load,
+			params,
+		})
 	}
-
-	/**
-	 * <https://github.com/CasparCG/help/wiki/AMCP-Protocol#MIXER-CLEAR>
-	 */
-	public mixerClear(channel: number, layer?: number): Promise<IAMCPCommand> {
-		return this.do(new AMCP.MixerClearCommand({ channel: channel, layer: layer }))
+	async play(params: PlayParameters): Promise<Request> {
```

Comment:
Now that the executeCommand isn't asynchronous, these methods shouldn't be either, right?